src/app.ts

```diff
@@ -19,6 +19,7 @@ interface Headers {
   'Content-Length': number;
   'Content-Type': string;
   'X-Aurora-Error-Code'?: string;
+  'X-Aurora-Error-Gas-Burned'?: string;
```

Comment:
Let's rename the header to `X-NEAR-Gas-Burned`, because:

1. It's not Aurora or EVM gas we're measuring here, but NEAR gas.
2. We're going to ultimately return this header not only on error conditions.